src/components/RepoPanel/components/RepoTile/RepoTile.tsx

```diff
@@ -1,40 +1,143 @@
-import {StarIcon} from 'components/icons';
-import React, {FC} from 'react';
-import {IUserRepoItem} from 'types';
-import './RepoTile.css';
-import dayjs from 'dayjs';
+import React, { FC } from "react";
+
+import { Skeleton } from "antd";
+import { Avatar } from "components";
+import { StarIcon } from "components/icons";
+import dayjs from "dayjs";
+import styled from "styled-components";
+import { IUserRepoItem } from "types";
 
 type RepoTileProps = {
   item: IUserRepoItem;
+  isLoading?: boolean;
+  onClick?: () => void;
```

Comment:
надо сделать так, чтобы `callback` принимал `id`
```suggestion
  onClick: (id: string) => void;
```
В таком случае, в компоненте выш вы будете знать на какой элемент нажали